tutorials/cyclecloud/install-cyclecloud.md

```diff
@@ -0,0 +1,105 @@
+# Install and Setup CycleCloud for a Lab environment
```

Comment:
Does examples/cycleserver_msi and examples/cycleserver deploy VNET and cycleserver automatically via a simple azurehpc config file.
It seems you are deploying the same but with all the manual steps?

Reply:
Yes, I'm using manual steps. These manual steps can be useful in scenarios: 
- where people may not want to install it using  azurehpc scripts; or 
- for learning purposes, where people wants to understand what exactly is installed/required.

I can add a mention about examples/cycleserver_msi and examples/cycleserver as an alternative option.